packages/e2e-playwright/models/issuer-list.ts

```diff
@@ -28,6 +28,23 @@ class IssuerList {
     async selectHighlightedIssuer(issuerName: string) {
         await this.highlightedIssuerButtonGroup.getByRole('button', { name: issuerName }).click();
     }
+
+    async typeToFilterTerm(filter: string) {
+        await this.selectorCombobox.focus();
+        await this.selectorCombobox.type(filter);
+    }
+
+    async pressKeyboardToNextItem() {
+        await this.selectorCombobox.press('ArrowDown');
```

Comment:
Couldn't we just simulate a global keyboard event here, without explicitly using the 'selectorCombobox' item? (Ex: `await keyboard.press('ArrowDown')`);

My point is that, in a real scenario, the keyboard is kinda 'global' and it just fires events, and if the Component is properly focused, it should receive those keyboard events and handle it. 

In this case, we are explicitly selecting the combobox. I expect that this function might work even if the Combobox isn't focused/active by the 'user' (I am assuming, I didn't have time to test it though!)